### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by j3qq4hch**
Brief description of what sadulli is with pictures provided

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
-vo SADULLI, in the kitchen garden grows parsley
+# Zubax Sadulli
 
-[Sadulli piccino](https://a360.co/2OxvigG)
+## Overview
 
-[Sadulli grosso](https://a360.co/2OwM1k0)
+Sadulli is a 500 W integrated drive for unmanned aerial applications that contains a motor and its control electronics in a compact enclosure. 
+
+The main design goal is to create a compact yet sophisticated motor controller that can be used with relatively small BLDC motors (about 40-45 mm in diameter) and can form an integrated drive together with the motor. This integrated drive should be possible to use as a direct replacement for just a motor alone without any major modifications to the aircraft it is used on.
```
Comment (dimracer) [inline README.md]: relatively small BLDC and PMSM motors

File: README.md [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+
+Sadulli is equipped with one CAN bus. The power is supplied through standard XT30 male connector.
+
+<img src="pics/Sadulli connectors drawing.png" alt="grosso" style="zoom:60%;" />
+
+Under the hood Sadulli represents a construction of three PCBs (excluding Mitochondrik)
+
+<img src="pics/Sadulli PCB.png" alt="grosso" style="zoom:60%;" />
+
+The main PCB that determines most of SADULLI properties is the power stage PCB. The power stage is composed from three [BUK9K6R2-40E](https://www.digikey.com/products/en?keywords=1727-7274-1-ND) MOSFET arrays.
```
Comment (dimracer) [inline README.md]: Sadulli

File: README.md [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+| Parameter                                               | Value               |
+| :------------------------------------------------------ | ------------------- |
+| Drain to Source Voltage (V<sub>ds</sub>)                | 40 V                |
+| Current - Continuous Drain (I<sub>d</sub>) @ 25°C       | 40 A                |
+| R<sub>ds on</sub> (Max) @ I<sub>d</sub>, V<sub>gs</sub> | 6 mOhm @ 25 A, 10 V |
+| Gate Charge (Q<sub>g</sub>) (Max) @ V<sub>gs</sub>      | 35.4 nC @ 10 V      |
+
+Current shunt value is 3 mOhm
+
+Bulk capacitor bank is formed with 6 [68µF  aluminum electrolytic capacitors](https://www.digikey.com/product-detail/en/w-rth-elektronik/860020673014/732-8860-3-ND/5727097) from Würth
```
Comment (dimracer) [inline README.md]: Würth Elektronik.

**Review by dimracer — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,10 +1,58 @@
...
+| Grosso  | [V4014](http://en.rcsunnysky.com/v-multi-rotorefficiencytype/1083.html) | 17                        | 6.2                    | 1.5                 | 4               | 207      |
+| Piccino | [V4006](http://en.rcsunnysky.com/v-multi-rotorefficiencytype/1081.html) | 15                        | 5.5                    | 0.7                 | 1.5             | 128      |
+| Nudo    | N/A                                                          | N/A                       | N/A                    | N/A                 | N/A             | 62       |
+
+<p align="center">
+<img src="pics/nudo.png" alt="nudo" width = "40%" />
+<p align="center">
+<img src="pics/piccino.png" alt="piccino"  width = "40%" />
+<p align="center">
+<img src="pics/grosso.png" alt="grosso"  width = "40%" />
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
<p align="center">
<img src="pics/nudo.png" alt="nudo" width="40%" />
<img src="pics/piccino.png" alt="piccino" width="40%" />
<img src="pics/grosso.png" alt="grosso" width="40%" />
</p>
```

Please make sure you close your tags properly.

File: README.md [hunk 5]
```diff
@@ -1,10 +1,58 @@
...
+Sadulli is a 500 W integrated drive for unmanned aerial applications that contains a motor and its control electronics in a compact enclosure. 
+
+The main design goal is to create a compact yet sophisticated motor controller that can be used with relatively small BLDC motors (about 40-45 mm in diameter) and can form an integrated drive together with the motor. This integrated drive should be possible to use as a direct replacement for just a motor alone without any major modifications to the aircraft it is used on. 
+
+## Brief specs
+
+- Operating voltage range 4-8S Li-Po (LiCoO<sub>2</sub>) battery (12-34 V)
+- Maximum continuous power 500 W
+- [UAVCAN hardware recommendations](https://uavcan.org/Specification/8._Hardware_design_recommendations/) following
+- Absence of necessity to invoke soldering to setup the device
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
- Solderless integration
```

File: README.md [hunk 6]
```diff
@@ -1,10 +1,58 @@
...
+## Overview
 
-[Sadulli grosso](https://a360.co/2OwM1k0)
+Sadulli is a 500 W integrated drive for unmanned aerial applications that contains a motor and its control electronics in a compact enclosure. 
+
+The main design goal is to create a compact yet sophisticated motor controller that can be used with relatively small BLDC motors (about 40-45 mm in diameter) and can form an integrated drive together with the motor. This integrated drive should be possible to use as a direct replacement for just a motor alone without any major modifications to the aircraft it is used on. 
+
+## Brief specs
+
+- Operating voltage range 4-8S Li-Po (LiCoO<sub>2</sub>) battery (12-34 V)
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
- Operating voltage range: 4-8S Li-ion LiCoO<sub>2</sub>, 12-34 V
```

File: README.md [hunk 7]
```diff
@@ -1,10 +1,58 @@
...
+The main PCB that determines most of SADULLI properties is the power stage PCB. The power stage is composed from three [BUK9K6R2-40E](https://www.digikey.com/products/en?keywords=1727-7274-1-ND) MOSFET arrays. 
+
+| Parameter                                               | Value               |
+| :------------------------------------------------------ | ------------------- |
+| Drain to Source Voltage (V<sub>ds</sub>)                | 40 V                |
+| Current - Continuous Drain (I<sub>d</sub>) @ 25°C       | 40 A                |
+| R<sub>ds on</sub> (Max) @ I<sub>d</sub>, V<sub>gs</sub> | 6 mOhm @ 25 A, 10 V |
+| Gate Charge (Q<sub>g</sub>) (Max) @ V<sub>gs</sub>      | 35.4 nC @ 10 V      |
+
+Current shunt value is 3 mOhm.
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
The current shunt value is 3 mOhm.
```

File: README.md [hunk 8]
```diff
@@ -1,10 +1,58 @@
...
+
+Bulk capacitor bank is formed with 6 [68µF  aluminum electrolytic capacitors](https://www.digikey.com/product-detail/en/w-rth-elektronik/860020673014/732-8860-3-ND/5727097).
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
The bulk capacitor bank is formed with 6 [68µF  aluminum electrolytic capacitors](https://www.digikey.com/product-detail/en/w-rth-elektronik/860020673014/732-8860-3-ND/5727097).
```

File: README.md [hunk 9]
```diff
@@ -1,10 +1,58 @@
-# Sadulli
-vo SADULLI, in the kitchen garden grows parsley
+# Zubax Sadulli
 
-[Sadulli piccino](https://a360.co/2OxvigG)
+## Overview
```
Comment (pavel-kirienko) [inline README.md]: I don't think we need this section here.

File: README.md [hunk 10]
```diff
@@ -1,10 +1,58 @@
...
-[Sadulli grosso](https://a360.co/2OwM1k0)
+Sadulli is a 500 W integrated drive for unmanned aerial applications that contains a motor and its control electronics in a compact enclosure. 
+
+The main design goal is to create a compact yet sophisticated motor controller that can be used with relatively small BLDC motors (about 40-45 mm in diameter) and can form an integrated drive together with the motor. This integrated drive should be possible to use as a direct replacement for just a motor alone without any major modifications to the aircraft it is used on. 
+
+## Brief specs
+
+- Operating voltage range 4-8S Li-Po (LiCoO<sub>2</sub>) battery (12-34 V)
+- Maximum continuous power 500 W
+- [UAVCAN hardware recommendations](https://uavcan.org/Specification/8._Hardware_design_recommendations/) following
```
Comment (pavel-kirienko) [inline README.md]: Please find better wording.

File: README.md (hunk 4 above)
Comment (pavel-kirienko) [reply README.md]: Also, the way the pics are arranged now is not great. Can we please put them in one row? You can use a table if you want, like I did here: https://github.com/Zubax/opencopcho#design

File: README.md [hunk 11]
```diff
@@ -1,10 +1,58 @@
...
+<img src="pics/nudo.png" alt="nudo" width = "40%" />
+<p align="center">
+<img src="pics/piccino.png" alt="piccino"  width = "40%" />
+<p align="center">
+<img src="pics/grosso.png" alt="grosso"  width = "40%" />
+
+Sadulli is equipped with one CAN bus. The power is supplied through standard XT30 male connector.
+
+<p align="center">
+<img src="pics/Sadulli connectors drawing.png" alt="grosso"  width = "100%" />
```
Comment (pavel-kirienko) [inline README.md]: Tags

File: README.md [hunk 12]
```diff
@@ -1,10 +1,58 @@
...
+  
+Under the hood Sadulli represents a construction of three PCBs (excluding Mitochondrik).
+
+<p align="center">
+<img src="pics/Sadulli PCB.png" alt="grosso"  width = "60%" />
```
Comment (pavel-kirienko) [inline README.md]: Tags

File: README.md [hunk 13]
```diff
@@ -1,10 +1,58 @@
...
+| Drain to Source Voltage (V<sub>ds</sub>)                | 40 V                |
+| Current - Continuous Drain (I<sub>d</sub>) @ 25°C       | 40 A                |
+| R<sub>ds on</sub> (Max) @ I<sub>d</sub>, V<sub>gs</sub> | 6 mOhm @ 25 A, 10 V |
+| Gate Charge (Q<sub>g</sub>) (Max) @ V<sub>gs</sub>      | 35.4 nC @ 10 V      |
+
+Current shunt value is 3 mOhm.
+
+Bulk capacitor bank is formed with 6 [68µF  aluminum electrolytic capacitors](https://www.digikey.com/product-detail/en/w-rth-elektronik/860020673014/732-8860-3-ND/5727097).
+
+Overcurrent protection trip value is set to ~47A using 1K resistor on OC_adj pin (R1 on the connector PCB).
```
Comment (pavel-kirienko) [inline README.md]:
```suggestion
The overcurrent protection is configured to trigger at ~47 A using a 1k resistor on the `OC_adj` pin (R1 on the connector PCB).
```
Articles, units. You need to be more careful about these things.

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
@j3qq4hch Okay. I think the rest will be covered in the datasheet. In the future we will be following a stricter policy for design documentation; this one is an exception because this product was designed before the policy was put in place. The same applies to MyDuck and others.

Starting with the next design, no PR can be accepted unless it is accompanied with extensive design documentation as described in the policy.

File: README.md (hunk 4 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 5 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 6 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 7 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 8 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 9 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 10 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 12 above)
Comment (j3qq4hch) [reply README.md]: *

File: README.md (hunk 13 above)
Comment (j3qq4hch) [reply README.md]: *

**Review by pavel-kirienko — APPROVED** [review]

**Comment by pavel-kirienko** [comment]
@dimracer твой ход

**Review by dimracer — APPROVED** [review]

**Pull request merged**